__tests__/app.test.js

```diff
@@ -0,0 +1,46 @@
+const seed = require("../db/seeds/seed.js");
+const app = require("../app.js");
+const request = require("supertest");
+const data = require("../db/data/test-data");
+const db = require("../db/connection.js");
+
+afterAll(() => {
+  db.end();
+});
+
+beforeEach(() => {
+  return seed(data);
+});
+
+describe("GET /api/topics", () => {
+  test("200, respond with array of objects with property of slug & description", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        const topicsArray = body.topics;
+
+        expect(topicsArray).toHaveLength(3);
+
+        topicsArray.forEach((topic) => {
+          expect(topic).toEqual(
+            expect.objectContaining({
+              slug: expect.any(String),
+              description: expect.any(String),
+            })
+          );
+        });
+      });
+  });
+  describe("404: end point not found", () => {
```

Comment:
This is good, although I would advise moving outside of the describe block on line 15. This 404 for an endpoint that doesn't exist is not specific to "GET /api/topics", it will cover any instance of a non-existant endpoint for your whole app 👍